src/components/PerPartySearchPanelContainer.js

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import { DESKTOP_MIN_WIDTH, media, DISPLAY_FONT } from "../styles"
+import PartyDropdown from "./PartyDropdown"
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import Unimplemented from "./Unimplemented"
+
+const sectionStyling = { borderBottom: "1px solid #000", padding: "10px 0" }
+
+// @todo #1 implement search and select with visualization of each party
+export default function PerPartySearchPanelContainer({
+  totalVote = "16123223",
+  totalDistrictCouncilor = 10,
+  totalPartyListCouncilor = 5,
+}) {
+  return (
+    <div
+      css={{
+        fontFamily: DISPLAY_FONT,
+        margin: "20px 0 0 0",
+        [media(DESKTOP_MIN_WIDTH)]: {
+          display: "block",
+          order: 1,
+          padding: 0,
+          width: "275px",
+        },
+      }}
+    >
+      <PartyDropdown partyId={partyId} dropdownOpen={false} />
+      <PartyTotalVote totalVote={totalVote} />
+      <PartyTotalCouncilorEstimationVisualization />
+      <PartyTotalCouncilorEstimationNumber
+        totalDistrictCouncilor={totalDistrictCouncilor}
+        totalPartyListCouncilor={totalPartyListCouncilor}
+      />
+      <PartyPresidentCandidateList />
+    </div>
+  )
+}
+
+function PartyTotalVote({ totalVote }) {
+  return (
+    <div css={{ ...sectionStyling }}>
+      <div>คะแนนเสียงทั้งประเทศ</div>
+      <div
+        css={{ fontWeight: "bold", fontSize: "2.2em", padding: "10px 0 0 0" }}
+      >
+        {totalVote.replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,")}
```

Comment:
We already have a function for this propose, https://github.com/codeforthailand/election-live/blob/master/src/util/format.js#L5. it would be good if you could use it.

Reply:
ok k. fixed.